CHANGELOG.md

```diff
@@ -5,12 +5,18 @@
 * [#515](https://github.com/atoum/atoum/pull/515) Fix PHP7 support in the basic resolver ([@hywan])
 * [#516](https://github.com/atoum/atoum/pull/516) Add a "callStaticOnTestedClass" method ([@mikaelrandy])
 
-# 2.3.0 - 2015-10-22
+## Bugfix
+
+* [#526](https://github.com/atoum/atoum/pull/526) Mock asserter is not case sensitive ([@mageekguy])
+
+
+# 2.3.0 - 2O15-10-22
```

Comment:
Why you haven't corrected this before merging ?

(fixed here : https://github.com/atoum/atoum/commit/12fa40738d417902738c9cd561868396523e8785)